content/post/050-ipfs-camp-recap.md

```diff
@@ -0,0 +1,87 @@
+
+![](https://user-images.githubusercontent.com/618519/60863676-5012e680-a1d6-11e9-9775-2a8fc1b014a3.png)
+
+Catch up on all the recordings at 📼 [https://github.com/ipfs/camp#%EF%B8%8F-lightning-talks](https://github.com/ipfs/camp#%EF%B8%8F-lightning-talks)
+
+#### [**🧬 Sci-Fi Fair**](https://github.com/ipfs/camp#-sci-fi-fair)
+> Get blown away by huge Sci-Fi experiments, soon to be seen in a Network close to you
+
+![](https://user-images.githubusercontent.com/618519/60863667-4db08c80-a1d6-11e9-9c9b-cf153977ac7f.png)
+
+Catch up on all the interviews at 📼 [https://github.com/ipfs/camp#-sci-fi-fair](https://github.com/ipfs/camp#-sci-fi-fair)
+
+#### [**🧩 Unconf**](https://github.com/ipfs/camp#-unconf)
+> Gather folks around shared topics of interest and discuss collaborations.
+
+![](https://user-images.githubusercontent.com/618519/60865107-2cea3600-a1da-11e9-9287-22587bfdf9cd.png)
+
+Catch up on all the artefacts at 📒 [https://github.com/ipfs/camp#-unconf](https://github.com/ipfs/camp#-unconf)
+
+### IPFS Camp `Success` is `defined` by the number of `friendships`, `conversations`, `collaborations` and `new projects` that get started.
+
+And it was a blast. Here is a quick video recap as a souvenir for everyone that was there, and a postcard for everyone that didn't get a chance this time around:
+
+[![](img/060-ipfs-camp-recap/ipfs-camp-youtube-preview)](https://youtu.be/kc_dxO-V8YM)
+
+However, the IPFS Camp spirit doesn't stop here, now it is all about continuing the dialog that got started and taking all the learnings and materials to local IPFS Communities, showing and teaching more folks how to build on the P2P Web. All the content produced can be found in the IPFS Camp Repo -- http://github.com/ipfs/camp. We will be posting all the recordings soon as well.
+
+Last, but definitely not the least, a huge THANK YOU ❤️ to everyone that contributed to the event. IPFS Camp wouldn't be possible if it werent for:
+
+- The **160 Attendees** coming from over **22 Countries**!
+- The [**67 Content Creators**](https://camp.ipfs.io/team) that produced all the training and talk material for the camp attendees
+- An outstanding [Production Team](https://camp.ipfs.io/team) that created the phenomenal experience for everyone
+- The over 4000 IPFS Community members that have been building this project with us since the beginning
+
+[![](https://media.giphy.com/media/mGW5xq4SwlqnSTxwtb/giphy.gif)](img/060-ipfs-camp-recap/camp-gif.gif)
+
+On behalf of the whole IPFS Project, thank you all for being part of this project and community, such an ambitious goal couldn't be achieved without all your energy, passion and dedication to make it happen. Onwards!
+
+David Dias & The IPFS Team
+
+`Extra:` As a closing gift, here are some random interesting facts about Camp:
+
+- 😸 IPFS Camp was the 1st time the Lazer Cat bot became [**P2P Lazer Cat bot**](https://github.com/gorhgorh/ipfscatremote)
```

Comment:
Small typo: "laser" stands for "Light Amplification by Stimulated Emission of Radiation" so is never spelt with a "z"